glib/Gir.toml

```diff
@@ -55,6 +56,37 @@ manual = [
 [[object]]
 name = "GLib.*"
 status = "generate"
+    [[object.function]]
+    name = "win32_locale_filename_from_utf8"
```

Comment:
Something I just thought about: if a method/function start with `win32_`, shouldn't we add a `windows` `cfg_condition` directly in gir? That would make such cases unneeded to be done.

Reply:
That would be nice indeed

Reply:
We merge as is and I send a PR to fix that?

Reply:
Sounds good

Reply:
it would be even cooler if once we detect the prefix it would move all the functions inside `win32` and `unix` modules so that you get `glib::win32::locale_filename_from_utf8`. 

(I agree this can be done in a follow up step)